recipes/diff_wfst/iamdb/README.md

```diff
@@ -0,0 +1,5 @@
+Scripts for preparing the IAMDB dataset can be found at `gtn_applications/datasets/iamdb.py` and preparing the wordpieces can be found at `gtn_applications/datasets/make_wordpieces.py`. 
+
+For preparing the pruned ngram transition graph, use `gtn_applications/scripts/build_transitions.py`. 
+
+
```

Comment:
Maybe we should add tables to this file which are organized according to the figures/tables in the paper and list the relevant jsons for each?

Reply:
Let me create a new PR updating README